Tribler/Core/Libtorrent/LibtorrentDownloadImpl.py

```diff
@@ -278,16 +294,15 @@ def network_create_engine_wrapper(self, pstate, initialdlstatus=None):
                 has_resume_data = resume_data and isinstance(resume_data, dict)
                 if has_resume_data:
                     atp["resume_data"] = lt.bencode(resume_data)
-                self._logger.info("%s %s", self.tdef.get_name_as_unicode(), dict((k, v)
-                                  for k, v in resume_data.iteritems() if k not in ['pieces', 'piece_priority', 'peers']) if has_resume_data else None)
```

Comment:
Because it logs a lot of data and some may unnecessary. Previously, I used that for debugging.